src/main/kotlin/uk/gov/dwp/dataworks/aws/AwsParsing.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.dwp.dataworks.aws
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.core.io.ClassPathResource
+import org.springframework.stereotype.Component
+import uk.gov.dwp.dataworks.AwsIamPolicyJsonObject
+import uk.gov.dwp.dataworks.logging.DataworksLogger
+import uk.gov.dwp.dataworks.services.ConfigurationResolver
+
+@Component
+class AwsParsing(){
+    @Autowired
+    private lateinit var  configurationResolver: ConfigurationResolver
+
+    @Autowired
+    private lateinit var awsCommunicator: AwsCommunicator
+
+    companion object {
+        val logger: DataworksLogger = DataworksLogger(LoggerFactory.getLogger(AwsParsing::class.java))
+    }
+
+    /**
+     * Helper method converts JSON IAM Policy to an instance of `AwsIamPolicyJsonObject` data class
+     * and inserts extra values before serialising back to JSON string.
+     */
+    fun parsePolicyDocument(pathToResource: String, sidAndAdditions: Map<String, List<String>>, statementKeyToUpdate: String): String {
+        val resource = ClassPathResource(pathToResource)
+        val mapper = ObjectMapper()
```

Comment:
Could we make this `mapper` a class scope val such that we don't create one every run?